poethepoet/config.py

```diff
@@ -51,6 +52,14 @@ def default_array_item_task_type(self) -> str:
     def global_env(self) -> Dict[str, str]:
         return self._poe.get("env", {})
 
+    @property
+    def envfile(self) -> Dict[str, str]:
+        if self._poe.get("envfile"):
+            envfile_path = Path(self._poe["envfile"]).resolve()
+            if envfile_path.exists() and envfile_path.is_file():
```

Comment:
```suggestion
            if envfile_path.is_file():
```
Unless I'm missing something?

Reply:
oh, I've always wondered about that. if `env_path.exist()` is unnecessary, then sure!